cfg/endpoints.js

```diff
@@ -1,6 +1,11 @@
 // server host url
 export const FORCE_REALTIME_HOSTNAME = null  // set to a specific hostname for local debugging
+export const SERVICE_HOSTNAME = 'api.trellus.ai'
 
 // realtime endpoints
 export const SUBSCRIBE_TRANSCRIPT_ENDPOINT = 'subscribe-transcript';
 export const SUBSCRIBE_COACHING_ENDPOINT = 'subscribe-coaching';
+export const SIGNUP_USER_ENDPOINT = 'signup-user'
+
+// extension ID
+export const EXTENSION_ID = "dpkajdodpdckkjhhdnpgmimdmceeilkc"
```

Comment:
does this mean we will no longer be able to test with locally built chrome extensions? looks like my current version has some rando extension id (kbpladmgjfebbmiehpfhdjblgcaklnfo)

Reply:
fixed